ships/augur.json

```diff
@@ -0,0 +1,4749 @@
+{
+  "description": "The Romulans built the Augur class battleships as rapid-response vessels to react to attacks on their territory. These sleek warbirds come packaged with side-mounted disruptors and added armor plates across their wingspan.",
+  "build_costs": {
+    "materials": [
+      {
+        "type": "crystal",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 1430
+      },
+      {
+        "type": "ore",
+        "grade": 3,
+        "rarity": "uncommon",
+        "value": 11000
+      },
+      {
+        "type": "ore",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 5400
+      }
+    ],
+    "others": [
+      {
+        "type": "3\u2605\u00a0Battleship\u00a0Parts",
+        "value": 135250
+      }
+    ],
+    "resources": [
+      {
+        "type": "tritanium",
+        "value": 4798000000
+      },
+      {
+        "type": "dilithium",
+        "value": 219500000
+      }
+    ]
+  },
+  "faction": "romulan",
+  "grade": 3,
+  "img_url": "./assets/augur.png",
+  "rarity": "epic",
+  "scrap_info": {
+    "scrapable": true
+  },
+  "ship_ability": {
+    "title": "Obliterate",
+    "description": "As long as the opponent's ship is Burning, if the opponent is hit with the Obliterator weapon, the AUGUR increases all weapon damage by 130.00% (cumulative)."
```

Comment:
For officers we added abilities like this as "buff". Do we want the same introduction here?

Reply:
Should be doable since I already have the code to detect it. As long as the trigger words are the same it should be able to detect that.